.github/workflows/test.yml

```
@@ -1,44 +1,86 @@
-   - name: Run formatter
-     run: nargo fmt --check
+  noir-version-list:
+    name: Query supported Noir versions
+    runs-on: ubuntu-latest
+    outputs:
+      noir_versions: ${{ steps.get_versions.outputs.versions }}
+
+    steps:
+      - name: Checkout sources
+        id: get_versions
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:2:87: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)